examples/decoding/plot_receptive_field.py

```diff
@@ -0,0 +1,124 @@
+"""
+=========================================
+Receptive Field Estimation and Prediction
+=========================================
+
+This example re-creates figures from a paper from the Lalor group which
```

Comment:
re-creates => reproduces

from a paper => from Lalor et al's mTRF toolbox in matlab [1]_